utils/lists.ts

```diff
@@ -442,6 +442,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Mysterious runes randomly generated and carved on chain. Gift from ancient world!",
     url: "https://runesmystery.github.io/"
   },
+ {
+    name: “sexy Loot - sLoots“,
```

Comment:
Please remove the double quotes and type them manually or copy what you done with the url